recipes/python-isal/meta.yaml

```diff
@@ -0,0 +1,44 @@
+
+{% set name = "isal" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "python-isal"  # To avoid confusion with isa-l proper
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 25828d73e71e17ffcc54aeda20b8e4ddb654e805fcd6dc471e6d51090ea8b860
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  # isa-l package only available on linux as of yet.
+  skip: True  # [not linux]  
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - isa-l >=2.26
+    - cython
+    - {{ compiler('c') }}
```

Comment:
This needs to be in `build:`, not `host`

Reply:
Thanks. Unfortunately this does not solve the problem with the libc.so and libpthread.so linking.

Reply:
I was able to fix it. Apparently only the c compiler needs to be in build, the rest needs to be in build. This is how similar packages (also build with cython) work. Thanks for the hint.